spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/RemoveOrphanFilesProcedure.java

```diff
@@ -49,7 +49,8 @@
       ProcedureParameter.optional("older_than", DataTypes.TimestampType),
       ProcedureParameter.optional("location", DataTypes.StringType),
       ProcedureParameter.optional("dry_run", DataTypes.BooleanType),
-      ProcedureParameter.optional("max_concurrent_deletes", DataTypes.IntegerType)
+      ProcedureParameter.optional("max_concurrent_deletes", DataTypes.IntegerType),
+      ProcedureParameter.optional("actual_files_table", DataTypes.StringType)
```

Comment:
What about renaming this to `file_list_view` or `listed_files_view`? I think that we want some identifier that matches the name of the action method; that's why I'm thinking `file_list` or `listed_files` that are similar to `compareToFileList`. But we also want to make it clear that this is a table or temporary view. I think it's more likely that the DF is registered as a temp view rather than stored in a table, so I'd probably use the suffix `_view`.